lib/xing/tasks/generate_dummy_api.rake

```diff
@@ -0,0 +1,56 @@
+namespace :dummy_api do
+
+  FIXED_ENDPOINTS = [
+    "/homepage",
+  ]
+
+  TARGET_DIRECTORY = File.join(File.dirname(Rails.root), 'dummy-api')
+
+  def routes
+    Rails.application.routes.url_helpers
+  end
+
+  desc "Wipe and regenerate API fixtures in /dummy-api from development mode endpoints"
+  task :regenerate => [ 'db:sample_data:reload', :wipe_dummy_directory, :generate ]
+
+  task :wipe_dummy_directory do
+    sh "rm -rf #{TARGET_DIRECTORY}"
+  end
+
+  desc "Generate API fixtures in /dummy-api from development mode endpoints"
+  task :generate => :environment do
+    require 'fileutils'
+
+    endpoints = FIXED_ENDPOINTS
+    puts "Making public pages"
```

Comment:
The FIXED_ENDPOINTS thing is related to snapshotting - likewise a target for later extraction